src/core/dev/cq_mgr_tx.cpp

```diff
@@ -135,6 +135,10 @@ bool cq_mgr_tx::request_notification()
 {
     cq_logfuncall("");
 
+    if (safe_mce_sys().doca_tx) {
```

Comment:
We need to move this to ring_simple. it will also save you L2 cache not accesing the cq mgr object.